tasks/steps/configure_instance.yml

```diff
@@ -10,6 +10,12 @@
         cartridge_defaults: '{{ cartridge_defaults }}'
       when: (restarted == false) or (restarted is none and not needs_restart)
 
+    - name: 'Check that cluster cookie is specified'
```

Comment:
Add assert to restart instnace step too

Reply:
In fact, it's needed only for `needs_restart` check.

Reply:
Yes, add before this check

Reply:
But it's checked inside by the module itself. Here cookie is used without any module, so this check can be performed only here.